### PR TITLE
chore: Remove experimental annotation where possible

### PR DESCRIPTION
**Pull request by joonas**
## Feature or Problem

This is no longer needed, so we should just remove it from our current sets of docs.

For older examples where it may make sense to leave it in for posterity, I put quotes around it.

### CODE REVIEW

File: community/2024-04-03-community-meeting.mdx [hunk 1]
```diff
@@ -31,7 +31,7 @@ metadata:
   annotations:
     version: v0.0.1
     description: 'HTTP hello world demo in Rust, using the WebAssembly Component Model and WebAssembly Interfaces Types (WIT)'
-    experimental: true
+    experimental: 'true'
```
Comment (vados-cosmonic) [inline community/2024-04-03-community-meeting.mdx:34]:
```suggestion
    experimental: "true"
```

Can we get consistent quotes? it's wild that we use both `'` and `"` all over the place

**Review by vados-cosmonic — APPROVED** [review]
LGTM :rocket:

File: community/2024-04-03-community-meeting.mdx (hunk 1 above)
Comment (joonas) [reply community/2024-04-03-community-meeting.mdx:34]: I was being consistent on a per example basis based on how the `description` is quoted, we aren't being consistent there across the board, so I figured it was more important to match that style

Comment (joonas) [reply community/2024-04-03-community-meeting.mdx:34]: I propose once we decide on the style we want to follow, we change this in a separate PR that addresses it across all of the examples with the new style in mind.

**Review by brooksmtownsend — APPROVED** [review]

**Pull request merged**